Create index.md (#10651)
PostHog/posthog.com: contents/handbook/growth/revops/index.md

---
title: Overview
sidebar: Handbook
showTitle: true
---

## How RevOps works
RevOps at PostHog is the Product Manager for Sales, Marketing, and Executive teams. Just as PMs help engineering teams build better products by connecting user needs with technical solutions, RevOps helps go to market teams make better decisions by connecting different parts of the business together.

> We do this by combining data from marketing, sales, product usage, and customer success to show what's working and what isn't. While individual teams deeply understand their specific areas, we provide insights about how different parts of the business affect each other and help teams see these connections to drive better outcomes.

Reviewer: ```suggestion
We do this by combining data from marketing, sales, product usage, and customer success to show what's working and what isn't. While individual teams deeply understand their specific areas, we provide insights about how different parts of the business affect each other and help teams see these connections to drive revenue growth for PostHog.
```

## RevOps values
1. Make data simple
2. Build for self service
3. Automate relentlessly

### 1. Make data simple
PostHog has data everywhere: product usage, sales pipelines, support tickets, revenue numbers. 
With this wealth of data comes complexity. We turn this scattered data into clear insights 
teams can use.

This means:
- Creating reliable, unified views that combine data from multiple sources
- Building clear abstractions that hide unnecessary complexity
- Maintaining source of truth definitions for key metrics
- Finding useful patterns in customer behavior
- Showing how each team's work affects our customers

> Creating views to show total monthly MRR, per product MRR, and per product usage, and filters out the anomalies to make sure make sure analyses are accurate and consistent across teams was one of the early steps we took in this direction. Unifying data from our billing system, salesforce, and vitally to have full context on biggest [gainers](https://us.posthog.com/project/2/insights/ZcynB70W) and [losers](https://us.posthog.com/project/2/insights/5V4mvAej) queries to show full context on which customers' spend changed the most was another one to simplify access to this info and quickly take action when needed.

Reviewer: ```suggestion
Creating views to show total monthly MRR, per product MRR, and per product usage, and filters out the anomalies to make sure make sure analyses are accurate and consistent across teams was one of the early steps we took in this direction. Unifying data from our billing system, Salesforce, and Vitally to have full context on biggest [gainers](https://us.posthog.com/project/2/insights/ZcynB70W) and [losers](https://us.posthog.com/project/2/insights/5V4mvAej) queries to show full context on which customers' spend changed the most was another one to simplify access to this info and quickly take action when needed.
```

before they leave.

### Things we don't want to spend time on
**Being the "data police":** We don't want to spend time enforcing data formats or policing how teams 
use tools. We focus on making it easy to do the right thing, not enforcing rules

**Running reports for people:** If someone regularly needs data, we should teach them how to get it themselves.

**Clean up projects:** If we're constantly cleaning up data problems, we've built the wrong systems, and should 
fix the source problems instead.

## Responsibilities

### What RevOps owns
Revenue insights:
- Reporting company wide metrics: revenue, retention, expansion, churn
- Help sales, marketing, and exec teams understand what drives revenue
- Identify patterns in customer behavior
- Build shared understanding of revenue and retention reporting across teams

Sales tech stack including:
- Salesforce administration and optimization
- Enrichment and intelligence tools (e.g. Clay, Clearbit, Sales Navigator)
- Contract management systems (Pandadoc)
[SalesOps section in handbook](https://posthog.com/handbook/growth/sales/crm) has more information.

### What RevOps supports but doesn't own

> Revenue reporting and forecasting: RevOps provides recommendations and improvements but does not own implementation and maintenance. This is currently owned by growth team. 

Reviewer: ```suggestion
Revenue reporting and forecasting: RevOps provides recommendations and improvements but does not own implementation and maintenance. This is currently owned by the Growth team. 
```

Marketing operations: Marketing owns their campaigns and analytics, we help connect marketing data with revenue outcomes.

Product operations: Product teams own their metrics and experimentation, but we help track how they impact overall revenue.

### What RevOps doesn't do
- Financial accounting (though we work closely with Finance)
- Individual sales deal management

> - Billing/invoicing platforms, and data infrastructure for revenue reporting owned by growth team

Reviewer: ```suggestion
- Billing/invoicing platforms, and data infrastructure for revenue reporting owned by the Growth team
```
